src/content/docs/en/guides/actions.mdx

```diff
@@ -18,6 +18,15 @@ Use actions instead of API endpoints for seamless communication between your cli
 - Generate type-safe functions to call your backend from the client and even [from HTML form actions](#call-actions-from-an-html-form-action). No need for manual `fetch()` calls.
 - Standardize backend errors with the [`ActionError`](/en/reference/modules/astro-actions/#actionerror) object.
 
+
+<Aside type='tip'>
+Actions don’t just send back simple JSON. Instead, they return a special kind of data package that can handle things like dates, maps, sets, and URLs. This is made possible using a tool called the **devalue library**. However, because of this special format, you can't easily inspect the response from the network like you might with regular JSON.  
+
+If you want to understand more about how this library works, you can check it out [Devalue](https://github.com/Rich-Harris/devalue).  
+
+For details about the `handler()` property mentioned, take a look at [Handler Property](/en/reference/modules/astro-actions/#handler-property).
```

Comment:
```suggestion
Actions return a custom data format that can handle Dates, Maps, Sets, and URLs [using the Devalue library.](https://github.com/Rich-Harris/devalue)

Because of this special format, you can't easily inspect the response from the network like you might with regular JSON. We recommend inspecting the `data` object returned by actions for debugging.

[See the `handler()` API reference](/en/reference/modules/astro-actions/#handler-property) for full details.
```

Reworded to clarify what users _should_ do when debugging. Also made the wording 10% more boring to quote @sarah11918 :)

I agree this isn't ideal, and I can try investigating other ways to give a preview of payloads in your network tab. May be a different `content-type` or a JSON object wrapper for the value.